kokoro/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/library/debian:bullseye AS build
+WORKDIR /app
+COPY . .
+RUN kokoro/rodete/fetch_dependencies.sh
+RUN rm -rf build
+RUN meson build
+RUN meson compile -C build
+RUN meson test --print-errorlogs -C build
+RUN meson install -C build
+
+FROM docker.io/library/debian:bullseye
+COPY --from=build /usr/local /usr/local
+COPY --from=build /app/kokoro/docker/glome-start /usr/local/sbin
+RUN apt-get update
+RUN apt-get -y install socat xxd libglib2.0-0
```

Comment:
These should usually be combined into one `RUN` block. 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Done, thanks!